parser/prism/Translator.cc

```diff
@@ -26,19 +26,23 @@ std::unique_ptr<parser::Assign> Translator::translateAssignment(pm_node_t *untyp
     auto node = reinterpret_cast<PrismAssignmentNode *>(untypedNode);
     auto *loc = &node->base.location;
 
-    auto name = parser.resolveConstant(node->name);
```

Comment:
I've moved this assignment to lines 34 and 45 so it only happens in conditional branches where `node->name` will exist. Is there a more elegant way to do this?

Reply:
That seems best IMO, because 1 of the 3 branches of the `if`/`else` ladder doesn't set it.